extensions/mssql/package.json

```diff
@@ -390,6 +399,36 @@
               "value": "serverEdition"
             }
           ]
+        },
+        {
+          "flavor": "isSqlOnDemand",
```

Comment:
>isSqlOnDemand [](start = 21, length = 13)

Flavor here should probably just be sqlOnDemon (or even on_demand) to match the other flavor names